include/boost/geometry/core/tags.hpp

```diff
@@ -105,6 +105,9 @@ struct box_tag : single_tag, areal_tag {};
 /// Convenience segment (2-points) identifying tag
 struct segment_tag : single_tag, linear_tag {};
 
+/// OGC Polyhedral surface identifying tag
+struct PolyhedralSurface_tag : single_tag, volumetric_tag {};
```

Comment:
But it should never contain capitals... I suggest `polyhedral_surface_tag`

Reply:
Ok, I will fix this.